src/components/sections/results/ResultCharacter.jsx

```diff
@@ -40,6 +40,9 @@ export default function ResultCharacter({ character, weapons, skills, assumedBuf
   const sigil2Id = allExtrasModifiersById[sigil2]?.gw2id;
   const rune = runeStringId ? allExtrasModifiersById[runeStringId] : undefined;
 
+  const isExotic = (index) =>
+    character.settings.cachedFormState.priorities.exotics.data?.[character.gear[index]]?.[index];
```

Comment:
```suggestion
    character.settings.cachedFormState.priorities?.exotics?.data?.[character.gear[index]]?.[index];
```

Not sure if this is the only place where this will occur, but due to the ability to load states created before this feature, we have to handle missing data without crashing.

(I tested this by just generating a link in the current branch, [such as this one.](http://localhost:3000/?m=raids&v=0&data=XQAAAALCNgAAAAAAAABAq4nnA9VsVEKPWfSDKzUnaifZ2LlbXZmA7WUgnMr3hgX6i0CzWo4Af-0B2ydvNkCRsFZEVhujNnqt4nkVKAsA-1tmNcIPdHsV7LmPX01hOHVtpHu01yF4zQAQlsY7F228K7x8KNNx7xtSNowv6cBbmAMBC29Ik2JzApC6480wI4zfCdOid2AYrReIszTRXDcGB-9JeZmgSqapZ3pdntskf7X3MR4vXtIotomfiBl1kUBbDWjb2XaqInDdW2GcPqrfzlgGEu3pt8JUR8-Q8sjCQR61BtIXDPU-T_5xWP1tWcWQhlxelGsf-0eL0XJKC-H1ESu_cuyC8LRIbvz3HKU7U7QY1IE96w6z7F2Sck10v0h2kiS0_fRL9bv-83y_I_OrD496974wHc7D6EgHiLdVisaFSdfi6hW5Ss61tAkTheo2LX_UkjDv51yGAolFIHwLUWduE7LDnkBjAhqgdzVt88ne6scsyncWxkmqKwe_5BCSHAgZsquv5tNhGaw-PgEU7aUTjr8RY-w7IEMesHEPVJ9OvP4XAdWZL_zlIFVhGUcQIl7cLMaK9i1Epxr1UebVP5ubACxK2NLB1fwxnYjgoZ4j2nbLXNSVlRFlT0rRpelZdxTtCWy9R3jjM12LQuQiEthMZxTOB0hesaE9wcP7uf3tuQ7GaURAv1Fl0T7ddUHlGDSqeaKSx3Vi4QoMy_rxFfxkkr3aawYyKAEYgm48TWxXbJPje9eubVX3E31kSeKrhu_7K8q1HbEbJtpqa6m0BOgigRlFJylojyMfpBF98LwTYJ7bSk1llhHBE-lc5F4YUu1vc6ccBP1koAPfN3b1UizXBG9JE3fT8PYxTTEFYfPZBwEA0ZXe7N3Fu5QI567cwAA1PlO8MAx-AbF9TIi5tLvHbslfAwovuQ91pFNCSBFDU8SdTz7o0fXP_KG5Ig_3bSNasu9fvehGXPOqdXQe23K8RU60LRwi2-bPqgHPGVECGE8UxceOQXNZTk7TgJv16Vb8U6oQTUdSohLq7UvL3eExuZD2w0jbRE97ala2rAWiClykmOEinL5jiNM82XyssCvOLtqwFgpPcw2j6Omg6HEopjbMq5gCczLRh6v3rlUIxNyQH3EHcEd589brH3S1-jcvEV54gcb4d36IKROdWl-3AdQNhm6uH1ewY80ZgCYMZd1p5LzWbEoXlB7UNUfMTdM2V7TRQANsA1ZFkqVYrNgn0EbTmsQIk5QpeuAY5AWHgMeaBZAyqJ-HF8Dx6IsYuNHVRGzSKf1GDjbeF1Lmfb5Aw1U8-vTmD6exiNMQrC7eLyE4SXMjWvpph1VuWYzg8WbAYL-bX2CYhs1DfmP0HXb8Jq-QnLt_6tLsDxKTIUHw-tk0kVcl-rcFN2wCsuENP1xl-HPI5U2jEdVOOhrYleAOcU2D1ItprrhT8CA8afvha77Zua3b4XFsNQNF15M5ZmUyiFZ-W1vjBF9LDnhKsdpxUG-evprh-YmXrnNMqSRE-r3CQvv00aLkADf_WPJseCSVYjL4Jgk9l7Mdw-mO3pJ5Znvvu0lsm00u8Cb5398BOGpbgtdEnbeRFuDito7A2sbbu3lzaMBclelF4Y87hzZ1eysQOptJDT_D0x1daYkkydB6t4zv_ciP5XeonMBJGfmUU2xNWn_HHUUCzHtrQgoMAOMeVks5rRxBOYQdMOAUFv9zNhtGEX6A3088KDYkNNn1dsMfwKWSBOho60M1bgOmWOBvF_WelI_v8VwRu1xNeKoz3GV_5NA7v2BXwonnIxWy6J01V8CFnhezuo1VL7RjTDfPG3wyqx91ZkjzWBScCSjUimJCnHjy0uCWH5f-_dB3mRo8a7ySBKlUKbgcny1AuOjlMOKG4kDjeNbcSXZdSLcj0iaHiAD2Hvoxpe3ryek7--dbp4yZaxx6NCi0q9P291WfmHr9PIW2255gm5BoJ4UmlOTXipml3XhcZyk3cioczAFf6O-1kwxJTXRWhtodrafwE9xU6O40vhZCCl1AO_F2V7Qt2heBhRHRyFVm9j9ysUfh-sOqZe1YvY2EAUvj1fNADjUXhnz9H-EPJU3ZwRuOdStvQdW4DLQ6Om8iiUSHYbAM9BmA2L-e4Fdvu6q39etXFb8dPk29eODSoGsgXY43CYUPbHB4ugLh9vU3yxlZXuaCM2nkkMIWji5W_3uN7CVW8ogKYEmLZu25ciK1TGYMgi-uwc3538P67Wq50-x7_x4O0dwM_6gQ1kb2N4Z4eLD0o2Z0HTf-6quWdauvqWVZHhlKFQ-1vmjDVRbSiOhIHqNfXMxUYPGdbzacPDCcxHTk1IJwsVQl061RviXCTcVBwRXB3iTcUKyroBv50UPbIPpsNyuR-7WH8ltnVd9-CUY5X4ExHOIwjZ15peIJenW21RB_o1X4lJXxnXyEfSKQlvIP3qHV7TSKwrK8v7wW1xTikidbQ2Om6IP-MDOT90xN4aPrN38mCQhIbuSpzFPuwKVjmc3IuswyYl0WmC8j2mvCifCAbrLRynROb4ozrbHTiedgPNFzvYHB5uM2w8P4FVl6qH9boQmORpK83qg09J2wtYhu30m8kpITXTpwnTV0MNd5mS_nAsJgN1_ywloXoe4K7IAr7FsK3S3NE0DeF4c_NnilH_i11ZjsrPUqZeJRhabYU4lC2a1Ao2jDdbqgrMNzPuH6iaiqGebx8bpCIZlKp0TQqcneMoSKJxYjgBbJByHm_KstE_pCrw8So2h-cqf7rw99IwGbLQqZQ8_Afy9An8ryuhsF6hO3WKvshjrAg3ZZVgtlXVUYG5MRhYkmzGqLj5se5L2yPu21J6byb_RkM_cn-_EzLMOpo8nkMEcSYQiWs7VKHz8W67rk9Np_Ce5LBOrRqe2ixv0m1EEjiAxbNXG71osGEtRIK-0ggCCr9nJhSqT7SXtrQzIvotnlvGS2y33XidZdpwI8QQR2qa5JPfFyAGo3V3NT1CzumyCsPYPqYScLZlsL9aKBMQXXT_HiXuucwBK4sFPaZc0wAo9sdlUbP91K8jxgFPPX2dGQ8kZrFBNHAcjnqvpx0qDNkA18_Y7cRXQrWv1OWdox9h01R1mdaX0JT8JC4qsR_a-y4E0uXosOQfmB0pPAxTRZJZTE6EY9i4J66TSRoREljhdAdDj7IuwXzX5mj5EnP1r9iRAAzBuyZtNupmHSBMRdJgMqtA5Fkq2uxaT-zcThyCc1eO52vugBkkoaLOSjNwVemBPYIoSk_m5Nd7KEA2PYadXP0s_QeqY3aM12wWUoeF5xAkO3UKLeYdZ98ndUl-TIqtx9rMrBE_2a6y4w76n3NLWVnlcihuJeOX739iQrVmem1NBPENf1J-3_aX2MX_Btghwe6owLYbgW9kLO3ZSY7SQC974tBq_r50tc2wpH1E29YIU19cT19p3PaR730froY_7uY97KxyBRVILB8DyguN7JI0IFNZuAafJu4PJRH8Pv5E23rVjwIz7iGmtw6kM52uo_23ZDhBeGJv_zivuh))